src/DotVVM.Framework/ResourceManagement/ScriptResource.cs

```diff
@@ -13,7 +13,7 @@ namespace DotVVM.Framework.ResourceManagement
     public class ScriptResource : LinkResourceBase, IPreloadResource, IDeferableResource
     {
         public bool Defer { get; }
-        public ScriptResource(IResourceLocation location, bool defer = false)
+        public ScriptResource(IResourceLocation location, bool defer = true)
```

Comment:
defer should be also set to true when using parameterless constructor

Reply:
Thanks, should be fixed.